drivers/overlay/overlay.go

```diff
@@ -839,6 +839,16 @@ func (d *Driver) get(id string, disableShifting bool, options graphdriver.MountO
 	if _, err := os.Stat(dir); err != nil {
 		return "", err
 	}
+	readWrite := true
+	// fuse-overlayfs doesn't support working without an upperdir.
```

Comment:
Not a blocker: we should add a note so that we don't forget to come back and remove this workaround later.